sfdx-project.json

```diff
@@ -3,63 +3,63 @@
         {
             "path": "./es-base-objects",
             "package": "ESObjects",
-            "versionName": "Base Objects - Winter '21",
-            "versionNumber": "50.0.0.NEXT",
+            "versionName": "Base Objects - Spring '21",
+            "versionNumber": "51.0.0.NEXT",
             "default": false
         },
         {
             "path": "./es-base-styles",
             "package": "ESBaseStylesLWC",
-            "versionName": "Base Styles LWC - Winter '21",
-            "versionNumber": "50.0.0.NEXT",
+            "versionName": "Base Styles LWC - Spring '21",
+            "versionNumber": "51.0.0.NEXT",
             "default": false
         },
         {
             "path": "./es-base-code",
             "package": "ESBaseCodeLWC",
-            "versionName": "Base Code LWC - Winter '21",
-            "versionNumber": "50.0.0.NEXT",
+            "versionName": "Base Code LWC - Spring '21",
+            "versionNumber": "51.0.0.NEXT",
             "default": false,
             "dependencies": [
                 {
                     "package": "ESObjects",
-                    "versionNumber": "50.0.0.LATEST"
+                    "versionNumber": "51.0.0.LATEST"
                 }
             ]
         },
         {
             "path": "./es-space-mgmt",
             "package": "ESSpaceMgmtLWC",
-            "versionName": "Space Management App - Winter '21",
-            "versionNumber": "50.0.0.NEXT",
+            "versionName": "Space Management App - Spring '21",
+            "versionNumber": "51.0.0.NEXT",
             "default": true,
             "dependencies": [
                 {
                     "package": "ESObjects",
-                    "versionNumber": "50.0.0.LATEST"
+                    "versionNumber": "51.0.0.LATEST"
                 },
                 {
                     "package": "ESBaseCodeLWC",
-                    "versionNumber": "50.0.0.LATEST"
+                    "versionNumber": "51.0.0.LATEST"
                 },
                 {
                     "package": "ESBaseStylesLWC",
-                    "versionNumber": "50.0.0.LATEST"
+                    "versionNumber": "51.0.0.LATEST"
                 }
             ]
         }
     ],
     "namespace": "",
     "sfdcLoginUrl": "https://login.salesforce.com",
-    "sourceApiVersion": "50.0",
+    "sourceApiVersion": "51.0",
     "packageAliases": {
         "ESObjects": "0Ho4W000000CaeJSAS",
         "ESBaseCodeLWC": "0Ho4W000000CaeTSAS",
         "ESBaseStylesLWC": "0Ho4W000000CaeOSAS",
         "ESSpaceMgmtLWC": "0Ho4W000000CaeYSAS",
-        "ESObjects@50.0.0-17": "04t4W000002PpiJQAS",
-        "ESBaseCodeLWC@50.0.0-17": "04t4W000002PpiOQAS",
-        "ESBaseStylesLWC@50.0.0-15": "04t4W000002PpiTQAS",
-        "ESSpaceMgmtLWC@50.0.0-11": "04t4W000002PpiYQAS"
+        "ESObjects@51.0.0-17": "04t4W000002PpiJQAS",
```

Comment:
You don't want to update those existing package versions.

Reply:
@pozil This was the alias updates! I thought with Spring 21 we want all the alias numbers to be release API version. Let me know if the build process does it.